tox.ini

```diff
@@ -56,12 +56,21 @@ deps =
   black~=22.0
 commands = black {posargs} '../rustworkx' '../tests' '../retworkx'
 
+[testenv:stubs]
+basepython = python3
+envdir = .tox/stubs
+deps =
+  mypy @ git+https://github.com/python/mypy.git
```

Comment:
We need to keep using the latest version until https://github.com/python/mypy/pull/14041 is out

Reply:
I think this was released as of Mypy 1.0 a couple of weeks ago.